app/notifications/process_notifications.py

```diff
@@ -138,8 +138,8 @@ def persist_notification(
         # Persist the Notification in the database.
         dao_create_notification(notification)
         if key_type != KEY_TYPE_TEST:
-            if redis_store.get(redis.daily_limit_cache_key(service.id)):
-                redis_store.incr(redis.daily_limit_cache_key(service.id))
+            if redis_store.get(redis.daily_limit_cache_key(service_id)):
+                redis_store.incr(redis.daily_limit_cache_key(service_id))
```

Comment:
while we are here, there is a logging statement on l144-145 in f-string style, perhaps update it?